<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidgetSpeaker</class>
 <widget class="QWidget" name="SettingsWidgetSpeaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="pteTestText">
     <property name="plainText">
      <string>Hello QtTextToSpeech,
this is an example text in English.

QtSpeech is a library that makes text to speech easy with Qt.
Done, over and out.
</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Volume:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slVolume">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>20</number>
       </property>
       <property name="value">
        <number>70</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rate:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slRate">
       <property name="minimum">
        <number>-10</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pitch:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slPitch">
       <property name="minimum">
        <number>-10</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Engine:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbEngines"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Language:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbLanguages"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Voice name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbVoices"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="btnSpeak">
       <property name="text">
        <string>Speak</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPause">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnResume">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Resume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutStatus">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
